src/ptp/ptpd2/protocol.c

```diff
@@ -2881,12 +2803,6 @@ handleManagement(MsgHeader *header, ssize_t length,
 		return;
 	}
 
-	if (isFromSelf) {
-		DBGV("handleManagement: Ignore message from self\n");
-		freeManagementTLV(&ptpInterface->msgTmp.manage);
```

Comment:
Check memory management implications.

Reply:
This is ok because the TLV storage does not get allocated if the message is looped back - the management message never gets unpacked. (We could do with tidying up some of the ptpd2 verbosity though!)